eng/native/configurecompiler.cmake

```diff
@@ -209,13 +209,15 @@ if (CLR_CMAKE_HOST_ARCH_AMD64)
 elseif (CLR_CMAKE_HOST_ARCH_I386)
   set(ARCH_HOST_NAME x86)
   add_definitions(-DHOST_X86)
+  add_definitions(-DHOST_32BIT)
 elseif (CLR_CMAKE_HOST_ARCH_ARM)
   if (ARM_SOFTFP)
     set(ARCH_HOST_NAME armel)
   else ()
     set(ARCH_HOST_NAME arm)
   endif ()
   add_definitions(-DHOST_ARM)
+  add_definitions(-DHOST_32BIT)
```

Comment:
It was not defined anywhere but still had a usage: https://github.com/dotnet/runtime/blob/6a4878096e63ccb54f73f8f17acf3762e50cf705/src/coreclr/vm/jitinterface.cpp#L10324